task-sdk/dev/generate_task_sdk_models.py

```diff
@@ -78,7 +78,7 @@ def generate_file():
 
     app = InProcessExecutionAPI()
 
-    latest_version = app.app.versions.version_values[-1]
+    latest_version = app.app.versions.version_values[0]
```

Comment:
This was a previous bug -- we want the most recent version (`[0]`) not the oldest version (`[-1]`)